RadixWallet/Features/DisplayEntitiesControlledByMnemonic/DisplayEntitiesControlledByMnemonic+Reducer.swift

```diff
@@ -10,21 +10,52 @@ public struct DisplayEntitiesControlledByMnemonic: Sendable, FeatureReducer {
 		public var deviceFactorSource: DeviceFactorSource { accountsForDeviceFactorSource.deviceFactorSource }
 
 		public let accountsForDeviceFactorSource: EntitiesControlledByFactorSource
-		public let displayRevealMnemonicLink: Bool
+		public var displayRevealMnemonicLink: Bool {
+			switch mode {
+			case .mnemonicCanBeDisplayed: true
+			case .mnemonicIsMissingNeedsImport: false
+			case .displayAccountListOnly: false
+			}
+		}
+
+		public var mnemonicIsMissingNeedsImport: Bool {
```

Comment:
I think `mnemonicIsMissing` or `mnemonicNeedsImport` is enough

Reply:
`mnemonicNeedsImport` is good